homeassistant/components/whirlpool/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Whirlpool Sixth Sense",
```

Comment:
No need to add a title if it's same as manifest name. It will automatically fall back to that.